src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenOperators.cs

```diff
@@ -13,6 +13,50 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
 {
     public class CodeGenOperatorTests : CSharpTestBase
     {
+
+        [Fact]
+        public void TestIsNullPattern()
```

Comment:
Also add couple tests were "is" is used as a condition - like:
```
if (c is null) . . .
```



---
In reply to: [118344562](https://github.com/dotnet/roslyn/pull/18657#discussion_r118344562) [](ancestors = 118344562)